08-components-lab/src/04-multicolor.jsx

```diff
@@ -14,8 +14,54 @@
  *
  */
 
-import tinycolor from 'tinycolor2';
+/**
+ * Created by ofek_am Vardi LTD. on 18/12/2016.
+ */
+/**
+ * Created by ofek_am on 18/12/2016.
+ */
 import React from 'react';
 
-export default React.createClass({
-});
+export default class MultiColor extends React.Component{
+
+    static propTypes = {
+
+    };
+
+    static defaultProps = {
+
+    };
+
+    constructor(props){
+        super(props);
+        this.onChange = this.onChange.bind(this)
+    }
+
+    state={
+        color:'green',
+        content:''
+    }
+
+    onChange(e){
+        debugger;
+        this.setState({color:e.target.value});
+    }
+
+    render(){
+        return <div>
+
+            <input type="color" onChange={this.onChange}/>
+            <br></br>
+            <div style={{background:this.state.color,filter:'brightness(140%)',height:50,width:50,display:'inline-block'}}/>
```

Comment:
Instead of setting colors manually please use tinycolor2 for color manipulation.
Also there is no reason to repeat all these divs. Consider creating array and mapping over it